core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -2333,6 +2333,25 @@ wasm_set_running_mode(WASMModuleInstance *module_inst, RunningMode running_mode)
     return set_running_mode(module_inst, running_mode, false);
 }
 
+bool
+wasm_runtime_instantiate_run_start_func(wasm_module_inst_t module_inst,
```

Comment:
this name is a bit confusing because it doesn't run wasi's entry point, `_start`.
how about wasm_runtime_execute_instantiate_functions?